ibc/host/identifiers.go

```diff
@@ -0,0 +1,100 @@
+package host
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+)
+
+const (
+	defaultIdentifierLength = 32
+	identifierPrefix        = "#"
+	identifierCharset       = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789._+-#[]<>"
+	defaultMaxIdLength      = 64
+	portMaxIdLength         = 128
+)
+
+// basicValidation performs basic validation on the given identifier
+func basicValidation(id string, minLength, maxLength int) error {
+	if strings.TrimSpace(id) == "" {
+		return coreTypes.ErrIBCInvalidID(id, "cannot be blank")
+	}
+
+	if len(id) < minLength || len(id) > maxLength {
+		return coreTypes.ErrIBCInvalidID(id, fmt.Sprintf("length must be between %d and %d", minLength, maxLength))
+	}
+
+	if !strings.HasPrefix(id, identifierPrefix) {
+		return coreTypes.ErrIBCInvalidID(id, fmt.Sprintf("must start with '%s'", identifierPrefix))
+	}
+
+	if strings.Contains(id, "/") {
```

Comment:
Move invalid chars (in case we need to add more) into a const alongside the others